### PR TITLE
dev/translation#4 - Refine upgrade steps for modifying nl_NL

### PR DESCRIPTION
**Pull request by totten**
Overview
----------------------------------------
This refines the upgrader step added in #17027.

CC @demeritcowboy @seamuslee001 

Before
----------------------------------------
* Upgrader step uses APIv4, which is likely to have reliability issues in future upgrades
    * See: https://docs.civicrm.org/dev/en/latest/framework/upgrade/#tips-prefer-simple-sql-semantics-over-apibaodao
* Upgrader step is missing `option_group_id` filter

After
----------------------------------------

* Upgrader uses SQL
* Upgrader includes `option_group_id` filter

### CODE REVIEW

File: CRM/Upgrade/Incremental/php/FiveTwentySix.php [hunk 1]
```diff
@@ -79,16 +79,16 @@ public static function addNLBEOptionValue(CRM_Queue_TaskContext $ctx) {
       'is_active' => 1,
     ]);
     // Update the existing nl_NL entry.
-    \Civi\Api4\OptionValue::update()
-      ->addWhere('name', '=', 'nl_NL')
-      // Adding check against label in case they've customized it, in which
+    $sql = CRM_Utils_SQL::interpolate('UPDATE civicrm_option_value SET label = @newLabel WHERE option_group_id = #group AND name = @name AND label IN (@oldLabels)', [
```
Comment (demeritcowboy) [inline CRM/Upgrade/Incremental/php/FiveTwentySix.php:82]: Maybe a silly question: Where is the interpolate function?

Comment (demeritcowboy) [reply CRM/Upgrade/Incremental/php/FiveTwentySix.php:82]: Oh I see - it's part of this patch.

Comment (totten) [reply CRM/Upgrade/Incremental/php/FiveTwentySix.php:82]: Yeah, the function was originally two lines and then got a little longer for silly reasons.

I could break it out into a separate PR if that seems tidier.

**Comment by demeritcowboy** [comment]
* General standards
    * [`r-explain`] __PASS__
    * [`r-user`] __PASS__
    * [`r-doc`] __PASS__
    * [`r-run`] __PASS__
      * Also tested while upgrading in non-English.
* Developer standards
    * [`r-tech`] __PASS__
    * [`r-code`] __PASS__
      * Normally the Utils::interpolate stuff would be separate but it's in its own commit.
    * [`r-maint`] __PASS__
      * Has frobnicate test.
    * [`r-test`] __PASS__

**Comment by eileenmcnaughton** [comment]
merging based on @demeritcowboy review

**Pull request merged**